requirements-dev.txt

```diff
@@ -15,3 +15,4 @@ ddt~=1.4.2
 qiskit-aer>=0.10.0
 pandas>=1.1.5
 cvxpy>=1.1.15
+pygsti
```

Comment:
Might want to fix it to the current version, in case syntax changes in the future